content/security.md

```diff
@@ -515,7 +515,162 @@ WebApp.connectHandlers.use(
 
 Helmet supports a large number of directives, users should further customise their CSP based on their needs. For more detail please read the following guide: [Content Security Policy](https://helmetjs.github.io/docs/csp/). CSP can be complex, so in addition there are some excellent tools out there to help, including [Google's CSP Evaluator](https://csp-evaluator.withgoogle.com/), [Report-URI's CSP Builder](https://report-uri.com/home/generate), [CSP documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src) from Mozilla and [CSPValidator](https://cspvalidator.org/).
 
-A great example of CSP and other Security Headers used in a Production Meteor Application has been provided by the [LegalRobot Team](https://gist.github.com/dhrubins/31248506873a006ba5e6b459a81855c2).
+The following example presents a potential CSP and other Security Headers used in a Production Meteor Application.
+This configuration may require customization, depending on your setup and use-cases.
+
+```javascript
+/* global __meteor_runtime_config__ */
+import { Meteor } from 'meteor/meteor'
+import { WebApp } from 'meteor/webapp'
+import { Autoupdate } from 'meteor/autoupdate'
+import { check } from 'meteor/check'
+import crypto from 'crypto'
+
+const self = '\'self\''
+const data = 'data:'
+const unsafeEval = '\'unsafe-eval\''
+const unsafeInline = '\'unsafe-inline\''
+
+/**
+ * Creates a helmet-compatible CSP-configuration
+ * @param externalHostUrls {Array|undefined} Optional array with allowed hosts
+ * @return {Object} a CSP configuration object
+ */
+export function createCSPOptions (externalHostUrls = []) {
```

Comment:
Would be better to include here the Helmet implementation than just a function to export.

Reply:
You are right, I am so used to separate these things from each other :-D

Reply:
I removed the functions and rewrote the code in a more linear fashion to make it much more understandable and finally call helmet at the end with the created options.